website/docs/API.md

```diff
@@ -170,15 +171,23 @@ Usually you should not need to call `unmount` as it is done automatically if you
 ### `debug`
 
 ```ts
-debug(message?: string): void
+
+interface DebugOptions { 
+  message?: string;
+  mapProps?: MapPropsFunction;
+}
+
+debug(options?: DebugOptions | string): void
 ```
 
-Pretty prints deeply rendered component passed to `render` with optional message on top.
+Pretty prints deeply rendered component passed to `render`. 
+
```

Comment:
Pls add lower-level headers for options as we do for other APIs

```suggestion
#### `message` option
```

Reply:
Very good idea I hadn't noticed this was done for other apis !